### ISSUE TITLE
Option to set own session handler

### ISSUE THREAD

**Issue by kukulich** [nette/tracy#154]
Tracy doesn't currently work if project runs on more than one server because of this commit: https://github.com/nette/tracy/commit/e1e204c6bc0cb81ada494161e89fb8ea380dc35b

I suggest to add method `setSession()` so it would be possible to set own session handler with shared storage.

**Comment by dg** [comment]
What means `doesn't work`?

**Comment by kukulich** [comment]
Page is generated by one server but request for tracy content is server by another server (because of proxy). However there's no tracy data on the second server because session storage is not shared among servers.

**Comment by kukulich** [comment]
I just tested it and it looks it still does't work.

**Comment by dg** [comment]
Your PHP session storage is shared among servers? Via shared directory or how?

**Comment by JanTvrdik** [comment]
usually redis / memcache / mysql / postgre

**Comment by Vrtak-CZ** [comment]
@dg via memcache

**Comment by kukulich** [comment]
Thanks, it works better now. However there may still be a problem. It works perfectly on one server but ajax requests don't work on our production with session shared via memcached. I'll try to investigate if it's Tracy problem or problem with our environment.

There's also a little BC break. In previous version Tracy can start session, now Tracy requires already started session.

**Comment by dg** [comment]
The fact that Tracy started session annoyed some people, so I created Tracy's own independent session storage. But this storage was not working on your shared servers. So I removed it. And now user must start session manually to see AJAX requests and redirects. 

Session can be started
1) before `Tracy\Debugger::enable()` 
2) after `Tracy\Debugger::enable()`, but you must call `Tracy\Debugger::dispatch()` after that
